packages/hardhat/test/Early.test.js

```diff
@@ -50,8 +50,7 @@ describe('Early mint', () => {
       await assertRevert(
```

Comment:
Is there a way to verify that the error is the right one? I mean, this error should be SaleStateError and it's not tested here. Would be nice to use this approach if possible in the tests below.

Reply:
Ref:
- https://github.com/Synthetixio/synthetix-v3/blob/main/packages/core-contracts/contracts/token/ERC20.sol#L18
-https://github.com/Synthetixio/synthetix-v3/blob/main/packages/core-contracts/test/contracts/token/ERC20.test.js#L177

Reply:
Okay, I will take a look at it.

Reply:
check now @slorenzo, modified the test using the method on synthetixio

Reply:
Thanks, looks good!